### PR TITLE
Scrollable chat area

### PR DESCRIPTION
**Pull request by Mari-Hayashi**
The chat area and conversation list are now scrollable.
The chat area is automatically scrolled to the bottom when the user loads the chat area and when a new message arrives. (So the newest message is visible to the user.)

### CODE REVIEW

File: src/client/src/Domains/Messages/GetMessages/index.tsx [hunk 1]
```diff
@@ -44,6 +44,14 @@ function Messages({ receiver }: MessagesProps) {
         });
     }, [receiver, user]);
 
+    // Scroll to the bottom of the chat area for every messages update.
+    React.useEffect(() => {
+        var messageBody = document.querySelector('#chatArea');
```
Comment (johan1505) [inline src/client/src/Domains/Messages/GetMessages/index.tsx]: var -> const

Comment (johan1505) [inline src/client/src/Domains/Messages/GetMessages/index.tsx]: I think "chatArea" would be a better name for the "messageBody" variable

**Review by johan1505 — CHANGES_REQUESTED** [review]
Can you attach or send me a picture/gif of how the message page looks now. Thanks.

File: src/client/src/Domains/Messages/GetMessages/index.tsx (hunk 1 above)
Comment (Mari-Hayashi) [reply src/client/src/Domains/Messages/GetMessages/index.tsx]: Done.

Comment (Mari-Hayashi) [reply src/client/src/Domains/Messages/GetMessages/index.tsx]: Done.

**Review by johan1505 — APPROVED** [review]

**Pull request merged**